crates/subspace-farmer/src/bin/subspace-farmer/main.rs

```diff
@@ -55,9 +50,23 @@ struct FarmingArgs {
     /// Disable farming
     #[clap(long)]
     disable_farming: bool,
+    /* ****** DSN parameters ******/
```

Comment:
Just a though, you can create a struct and flatten it instead of using C-style comments :slightly_smiling_face: